### DISCUSSION TITLE
[Solved] Osu!Lazer crashes on start :(

### DISCUSSION THREAD

**Discussion by vavalth** [ppy/osu#14110] [Q&A] [answered]
hello there, I hope everyone is doing fine.
Going straight to the subject : My osu!Lazer crashes when I launch it. I can see that it loads a little bit (the FPS etc in the corner right pops up and start to rumble) but after 1 sec it freezes and crashes. 
I've try to restart my Pc doesn't help. 
I did not try to reinstall the osu!Lazer , because I'm a bit scared to loose all the tracks I downloaded. The "normal" Osu! launches whiteout a problem, but ... after playing with Lazer it's not the samel lol and I don't have all of the tracks anyways. 

I looked a bit on google , I did not found a lot of stuff (it's really only Lazer, normal one goes fine) i try'd to delete the osu.cb (i think) and restart the program but did not change anything. 

Ok , so I would love if someone could tell me how I could save all my tracks so I can put everything back in a click (or two lol) and if this is possible I guess I will try to remove osu!Lazer and reinstall it (or repair it if it has that option ?) Hope it's not a biggy to do so. 

Voilà, ... If you need any kind of information about the problem let me now. I played for the last time yesterday around the same hour and their was no problem. I did add some new tracks yesterday and I did not close and open Lazer again till now, could it be a track that makes Lazer go brrr ? 

Thanks in advance for any advice and help !! Thanks a lot for the time and efforts you putted in Lazer I really enjoy all the little extra's it has to give.

**Comment by bdach** [comment, marked as answer]
Please attach logs from either:
* `%AppData%/osu/logs` (on Windows),
* `~/.local/share/osu/logs` (on Linux and macOS),
* `Android/Data/sh.ppy.osulazer/logs` (on Android).

**Reply by vavalth** [reply]
[runtime.log](https://github.com/ppy/osu/files/6919558/runtime.log)
[session.log](https://github.com/ppy/osu/files/6919559/session.log)
[update_success.log](https://github.com/ppy/osu/files/6919560/update_success.log)

**Reply by bdach** [reply]
Those are not logs from lazer, they are from stable. Please double-check that you've tried the correct directory.

**Reply by vavalth** [reply]
Hello , I don't have any log files in the osu!Lazer files. Only this (see attachement) but I really doubt it is what you need lol. 
[SquirrelSetup.log](https://github.com/ppy/osu/files/6926520/SquirrelSetup.log)
![osulazer](https://user-images.githubusercontent.com/18542953/128072200-657e23ea-849e-4bc7-ad0f-658064b25763.jpg)
I'm I doing something wrong ?

**Reply by bdach** [reply]
That's the *local* app data folder. Try `Roaming`. If there is no osu! folder there then please attach event viewer logs.

**Reply by vavalth** [reply]
You're a real chief ! 
[database.log](https://github.com/ppy/osu/files/6926698/database.log)
[performance.log](https://github.com/ppy/osu/files/6926699/performance.log)
[runtime.log](https://github.com/ppy/osu/files/6926700/runtime.log)
[updater.log](https://github.com/ppy/osu/files/6926702/updater.log)
Thank you for your time !!

**Reply by bdach** [reply]
The logs are showing that you have some outdated rulesets installed (hishigata, hitokori, mvis, rush, sentakki, swing, tau). Please try removing/updating them and seeing if that helps.

If it doesn't, please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Reply by vavalth** [reply]
Hello, thank you a lot, indeed it probably was something like that I removed most of them lol and the game goes smoothly, thank you a lot kind stranger !!
